program/calculate-compound-interest/CalculateCompoundInterest.fs

```diff
@@ -0,0 +1,6 @@
+let compoundInterest principal interest time : float =
+  principal * (1.0 + interest) / time
```

Comment:
need to check the formula here

Reply:
@victoriacheng15 can you retest

Reply:
It should be `1610.51` but I got `161051000`
![image](https://user-images.githubusercontent.com/35031228/234021349-dfc89c4a-c187-4e09-acaa-bd1fe763ec13.png)

Reply:
@victoriacheng15 can you retest now